c2corg_api/views/validation.py

```diff
@@ -90,7 +112,7 @@ def validate_preferred_lang_param(request, **kwargs):
     is correct ("...?pl=...").
     """
     lang = request.GET.get('pl')
-    validate_lang_(lang, request)
+    validate_pref_lang_(lang, request)
```

Comment:
Are you sure this change is needed? Do we really have cases where we pass a list of langs as ``pl``?